pkg/controller/windowsmachineconfig/tracker.go

```diff
@@ -11,7 +11,7 @@ import (
 	nc "github.com/openshift/windows-machine-config-operator/pkg/controller/windowsmachineconfig/nodeconfig"
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
 	"github.com/pkg/errors"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Note to myself: Remove `v1` alias

Reply:
PS: I really would like to figure out why this keeps getting added and stop that from happening.

Reply:
It should come from go-imports but in vscode, I changed to gofmt but I see this happening. IDK why either :(